src/bolos/cx_aes.c

```diff
@@ -1,13 +1,13 @@
-#include <assert.h>
 #include <err.h>
+#include <assert.h>
```

Comment:
Why did you change the order of the headers? `clang-format` sorts the headers alphabetically, by default. Did you use another option to sort them differently?

Reply:
No reason...